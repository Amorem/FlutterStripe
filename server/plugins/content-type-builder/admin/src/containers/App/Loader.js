import React from 'react';
import LoadingIndicator from 'components/LoadingIndicator';
import styles from './styles.scss';

const Loader = () => (
  <div className={styles.app}>
    <div className={styles.centered}>
      <LoadingIndicator />
    </div>
  </div>
);

export default Loader;
