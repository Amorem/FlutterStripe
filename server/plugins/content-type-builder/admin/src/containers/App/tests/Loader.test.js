import React from 'react';
import renderer from 'react-test-renderer';

import Loader from '../Loader';

describe('<Loader />', () => {
  it('should not crash', () => {
    const tree = renderer.create(<Loader />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
