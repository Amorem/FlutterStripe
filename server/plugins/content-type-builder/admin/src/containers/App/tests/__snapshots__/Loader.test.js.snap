// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should not crash 1`] = `
<div>
  <div>
    <div>
      <div />
    </div>
  </div>
</div>
`;
